app/components/works/show/attached_file_component.rb

```diff
@@ -18,7 +18,8 @@ def initialize(attached_file:, depth:)
       def path
         return preservation_path(attached_file) if in_preservation?
 
-        rails_blob_path(file, disposition: 'attachment')
+        file.save unless file.persisted?
+        rails_blob_path(file.blob, disposition: 'attachment')
```

Comment:
Did some ActiveStorage APIs change on us here?

Reply:
Yup, I could revert and highlight the error I got, but I worked through the code and realized it needed the blob specifically here.